### PR TITLE
Document Rollups SDK

### LINKED ISSUE #59: Add documentation regarding the Rollup SDK
The Rollup SDK documentation is here:
https://consensys.github.io/sumo/

We should probably add some conceptual information and perhaps a Howto/Tutorial.

Think about what the user needs to know about the SDK to get started using it.

### PR DESCRIPTION
**Pull request by rolandtyler**
# Pull request checklist

## Describe the change
add documentation on Rollups SDK

## Issue fixed
Fixes #59 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-on-read-the-docs)
  and added a [preview link](#preview).

### CODE REVIEW

**Review by alexandratran — APPROVED** [review]

**Review by bgravenorst — CHANGES_REQUESTED** [review]
We should add some concept information about this. What does the SDK allow you to do? 
In some of the Concept information about money orders or the workflows, how does the SDK fit into that?

How do you add this to your project? For example, the Besu plugin API you need to ensure your project gradle file contains certain information etc...

File: docs/Reference/SDK.md [hunk 1]
```diff
@@ -7,3 +7,28 @@ Description: Rollups SDK
...
+## Prerequisites
+
+* MacOS or Linux based system (the SDK is not supported on Windows)
+* [Docker](https://docs.docker.com/get-docker/)
+* [Java JDK](https://www.oracle.com/java/technologies/downloads/) (11 or greater)
+
+## Configuration
+
+Configure Docker to use the ConsenSys [registry](https://help.cloudsmith.io/docs/docker-registry#setup).
+If you need access to the registry, contact us on the [ConsenSys Rollups Discord channel](https://discord.gg/9mCVSY6).
```
Comment (NicolasMassart) [inline docs/Reference/SDK.md:20]: this points me to an archived staking channel. Is this the right invite link?

**Review by NicolasMassart — APPROVED** [review]

**Comment by bgravenorst** [comment]
@rolandtyler, could you perhaps check if https://github.com/ConsenSys/sumo/pull/2256 also requires any additional work in this PR?

File: docs/Reference/SDK.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Configure Docker to use the ConsenSys [registry](https://help.cloudsmith.io/docs/docker-registry#setup).
+If you need access to the registry, contact [ConsenSys support](https://consensys.net/contact/).
+
+Ensure the [Gradle build file](https://github.com/ConsenSys/sumo/blob/main/manager/web3j-sdk-examples/build.gradle)
+contains the `https://dl.cloudsmith.io/public/consensys/maven/maven/` repository
+and the `net.consensys:web3j-sumo:0.4.0` dependency.
+
+## Example
+
+[This example](https://github.com/ConsenSys/sumo/blob/main/manager/web3j-sdk-examples/src/main/java/net/consensys/sumo/web3j/sdk/example/Web3jSdkExample.java)
```
Comment (alexandratran) [inline docs/Reference/SDK.md:28]:
```suggestion
[This example](https://github.com/ConsenSys/sumo/tree/main/sdk/web3j-sdk-examples)
```

This link no longer works. I replaced with a link to the example directory with the useful readme.

File: docs/Reference/SDK.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+demonstrates how to use the SDK by performing the following actions:
+
+* Create accounts
+* Retrieve account IDs
+* Request money order creation
+* Redeem money order
```
Comment (alexandratran) [inline docs/Reference/SDK.md:34]:
```suggestion
demonstrates how to use the SDK by:

* Creating accounts.
* Retrieving account IDs.
* Requesting money order creation.
* Redeeming money orders.
```

**Comment by alexandratran** [comment]
@bgravenorst looks like https://github.com/ConsenSys/sumo/pull/2174 should be documented on the JSON-RPC API methods page (see https://github.com/ConsenSys/doc.rollups/issues/68).

**Pull request closed without merging**